tests/test_driver.py

```diff
@@ -303,7 +316,9 @@ async def run_test(
     starttime = datetime.now()
     home = Path(tempfile.mkdtemp(prefix="fishtest-", dir=tmp_root))
     test_env = makeenv(script_path, home)
-    if test_file_path.endswith(".fish"):
+    if IS_CYGWIN and is_tmux_test(test_file_path):
+        return TestSkip(arg)
```

Comment:
nice; we could also use this TestSkip approach for the FISH_CI_SAN workaround in future.
(Though I'll try to set aside some time to fix the tmux tests soon.. no need to spend too much time on them)